lib/Dialect/TorchConversion/Transforms/Passes.cpp

```diff
@@ -64,14 +72,16 @@ void mlir::torch::registerTorchConversionPasses() {
 
 void TorchConversion::createTorchBackendToLinalgOnTensorsBackendPipeline(
     OpPassManager &pm) {
-  // We want to fuse quantized operations together before lowering to linalg.
+  // We want to fuse quantized operations together before lowering to
+  // linalg.
   pm.addNestedPass<func::FuncOp>(Torch::createFuseQuantizedOpsPass());
   pm.addNestedPass<func::FuncOp>(Torch::createScalarizeShapesPass());
 
   // Lower to linalg + guards which is the input to codegen backends.
-  // We do this first as it tends to involve pattern-matching against constants,
-  // (e.g. dimensions which must be constant in a ranked programming model)
-  // and those constants get somewhat obscured by TorchToArith.
+  // We do this first as it tends to involve pattern-matching against
+  // constants, (e.g. dimensions which must be constant in a ranked
+  // programming model) and those constants get somewhat obscured by
+  // TorchToArith.
   pm.addNestedPass<func::FuncOp>(createConvertTorchToTMTensorPass());
```

Comment:
I think I have to expose `populatePatterns` for these passes too so that I can call these in between `tosa and linalg` passes if needed, but I have to explore this more.